src/tests/tests.fsx

```diff
@@ -0,0 +1,37 @@
+#r "System.Threading.dll"
```

Comment:
If we're using this script to compile tests in netcore, we can remove src/tests/files.fsx, right?